releasenotes/notes/accept-generators-31f080871015233c.yaml

```diff
@@ -0,0 +1,11 @@
+---
+features:
+  - |
+    The following meths now support iterators as inputs, in addition to
+    the existing support for lists:
```

Comment:
Technically I think these methods supported any sequence type not just lists. But that's me being overly pedantic it's not important or worth updating here.

Reply:
I rephrased it as sequences and generators because that should cover most of what people care about